customer/src/test/java/no/unit/nva/customer/events/handler/CustomerTableDynamodbStreamToEventBridgeHandlerTest.java

```diff
@@ -0,0 +1,359 @@
+package no.unit.nva.customer.events.handler;
+
+import static no.unit.nva.customer.model.PublicationInstanceTypes.ARTISTIC_DEGREE_PHD;
+import static no.unit.nva.customer.model.PublicationInstanceTypes.DEGREE_BACHELOR;
+import static no.unit.nva.customer.model.PublicationInstanceTypes.DEGREE_LICENTIATE;
+import static no.unit.nva.customer.model.PublicationInstanceTypes.DEGREE_MASTER;
+import static no.unit.nva.customer.model.PublicationInstanceTypes.DEGREE_PHD;
+import static no.unit.nva.customer.model.PublicationInstanceTypes.OTHER_STUDENT_WORK;
+import static no.unit.nva.customer.model.channelclaim.ChannelConstraintPolicy.EVERYONE;
+import static no.unit.nva.customer.model.channelclaim.ChannelConstraintPolicy.OWNER_ONLY;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsIterableContainingInAnyOrder.containsInAnyOrder;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import com.amazonaws.services.lambda.runtime.events.DynamodbEvent;
+import com.amazonaws.services.lambda.runtime.events.models.dynamodb.AttributeValue;
+import com.amazonaws.services.lambda.runtime.events.models.dynamodb.StreamRecord;
+import java.net.URI;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Random;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import no.unit.nva.customer.events.model.ChannelClaim;
+import no.unit.nva.customer.events.model.ChannelClaim.Constraints;
+import no.unit.nva.customer.events.model.ResourceUpdateEvent;
+import no.unit.nva.customer.model.PublicationInstanceTypes;
+import no.unit.nva.customer.model.channelclaim.ChannelConstraintPolicy;
+import no.unit.nva.stubs.FakeContext;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class CustomerTableDynamodbStreamToEventBridgeHandlerTest {
+
+    private static final Set<PublicationInstanceTypes> DEFAULT_SCOPE = Set.of(
+        DEGREE_PHD,
+        DEGREE_MASTER,
+        DEGREE_BACHELOR,
+        DEGREE_LICENTIATE,
+        ARTISTIC_DEGREE_PHD,
+        OTHER_STUDENT_WORK
+    );
+
+    private CustomerTableDynamodbStreamToEventBridgeHandler handler;
+
+    @BeforeEach
+    void beforeEach() {
+        handler = new CustomerTableDynamodbStreamToEventBridgeHandler();
+    }
+
+    @Test
+    void shouldHandleEmptyBatch() {
+        var event = new DynamodbEvent();
+        event.setRecords(Collections.emptyList());
+
+        assertDoesNotThrow(() -> handler.handleRequest(event, new FakeContext()));
+    }
+
+    @Test
+    void shouldEmitEventForNewChannelClaimAddedOnCreationOfNewCustomer() {
+        var channelIdentifier = UUID.randomUUID();
+        var channelId = channelId(channelIdentifier);
+        var customerIdentifier = UUID.randomUUID();
+        var organizationId = randomOrganizationId();
```

Comment:
Create a value-holding record that initialises with this state; re-use in all tests.